Resources/Prototypes/_Moffstation/Body/Prototypes/resomi.yml

```
@@ -9,7 +9,6 @@
       - torso
       organs:
         brain: OrganHumanBrain
-        eyes: OrganResomiEyes
```

```suggestion
        eyes: OrganHumanEyes
```
I am not sure if there's any mechanical effect, but all other species without special eyes get human eyes.

---

I think this was from me testing having Nightvision attached as a component for the eye. But since we don't have shitmed it doesn't work that way.